src/ClientGenerator/ClientGenerator.cs

```diff
@@ -102,6 +105,28 @@ private static bool CreateGrainClientAssembly(CodeGenOptions options)
         }
 
         /// <summary>
+        /// Configure logger factory for your client generator
+        /// TODO: fix logging in ClientGenerator, I'm not sure its logging worked properly before, taken that LogManager's state doesn't serialize well across AppDomain
+        /// </summary>
+        public ILoggerFactory LoggerFactory { get; set; } = CreateDefaultLoggerFactory("ClientGenerator.log");
```

Comment:
Is this property accessed somewhere else? otherwise just create the factory at the place you need it

Reply:
this is for user to configure their logger factory used for buildetime-codegen. ofc they can set it through constructor. But I was thinking having a setter might be better and more flexible

Reply:
Not sure I follow the expected usage, since this is invoked as an `.exe` by MSBuild

Reply:
I'm not entirely sure how it is invoked. let's sync up.